conda-recipe/meta.yaml

```diff
@@ -79,12 +79,12 @@ outputs:
         # additional dependencies for convenience in conda-forge
         - fsspec
         - pooch >=1.3.0
-        - zarr
+        - zarr >=2.12.0
 
       run_constrained:
         - pyside2 >=5.14.2,!=5.15.0
-        - pyqt >=5.12.3,!=5.15.0
         - pyside6 <6.3.2
+        - pyqt >=5.12.3,!=5.15.0,!=6.6.1
```

Comment:
Note [pyqt6 doesn't exist yet on conda-forge](https://github.com/conda-forge/pyqt-feedstock/pull/136), but for completeness, I'm adding it here.